### ISSUE TITLE
The accuracy

### ISSUE THREAD

**Issue by Ramesii** [unitreerobotics/unifolm-vla#2]
I have tested as below:
01/30 [15:20:01] INFO     | >> Arguments: {                                                                                                                                                                                                                            eval_libero.py:168
                              "task_suite_name": "libero_object",
                              "num_steps_wait": 10,
                              "num_trials_per_task": 50,
                              "window_size": 1,
                              "video_out_path": "results/libero_object/UnifoLM-VLA-Libero/checkpoints",
                              "local_log_dir": "./experiments/logs",
                              "seed": 42,
                              "pretrained_path": "/UnifoLM-VLA-Libero/checkpoints/pytorch_model.pt",
                              "post_process_action": true,
                              "unnorm_key": "libero_object_no_noops",
                              "vlm_pretrained_path": "/UnifoLM-VLM/"
                          }
But it cannot reach the very high rating,it can only obtain 

INFO     | >> Final results:                                                                                                                                                                                                                          
eval_libero.py:118
                 INFO     | >> Total episodes: 500                                                                                                                                                                                                                     eval_libero.py:118
                 INFO     | >> Total successes: 423                                                                                                                                                                                                                    eval_libero.py:118
                 INFO     | >> Overall success rate: 0.8460 (84.6%) 

maybe something wrong?

**Comment by LanrenzzzZ** [comment]
The released checkpoints were trained with window_size = 2, so evaluation should use the same setting.
Evaluating with window_size = 1 would require retraining the model.
In our experiments, we found that incorporating historical information leads to better performance.
